src/ert/shared/models/multiple_data_assimilation.py

```diff
@@ -119,12 +119,12 @@ def runSimulations(
                 ensemble_id=ensemble_id,
             )
             self.ert().runWorkflows(HookRuntime.POST_UPDATE)
+            _, ensemble_id = self._simulateAndPostProcess(
```

Comment:
Why `_, ensemble_id =`?

Reply:
Just moved that code, the `_` is the number of completed. Not quite sure what the question is?